lib/backlogs_hooks.rb

```diff
@@ -16,10 +16,10 @@ def exception(context, ex)
       def helper_issues_show_detail_after_setting(context={ })
       	begin
           if context[:detail].prop_key == 'release_id'
-            r = RbRelease.find_by_id(context[:detail].value)
+            r = RbRelease.find_by_id(context[:detail].value) unless context[:detail].value.include?('.')
```

Comment:
sometime it is an integer and sometimes a string, couldn't quite figure out when each occurs, i'll try the regex.